src/contexts/dictionary-provider/dictionary-provider.tsx

```diff
@@ -25,12 +27,12 @@ export const DictionaryProvider: FC<DictionaryProviderProps> = ({
 }
 
 export const useDictionary = () => {
-  const dictionary = useContext(DictionaryContext)
+  const xxx = useContext(DictionaryContext)
```

Comment:
why `xxx`?

Reply:
At that time, I temporarily changed it because I was studying and I forgot.